### DISCUSSION TITLE
Dependency Dashboard checkbox is unclickable

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#23738] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

The checkbox is clickable
https://github.com/Zlopez/anitya-renovate-poc/issues/5

### What you tried so far.

There was a renovate app configured in parallel to a self-hosted runner one. A [PR](https://github.com/Zlopez/anitya-renovate-poc/pull/8) was created yesterday by the renovate app.
The PR was closed today. I'd like a new PR generated by the self-hosted runner. The checkbox is not clickable.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
missing rights?

**Reply by lmilbaum** [reply]
Ahh. That could be it.
